src/Game.java

```diff
@@ -1,45 +1,119 @@
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.Scanner;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public class Game {
-    public static void main(String[] args) {
-        System.out.println("Welcome to PPTHangman!");
```

Comment:
Firstly, we tend to stick to 4-space indentation in Java, not 2.
Secondly, this is a perfect example of what is called "commit/diff noise". These are essentially changes which do not represent any logical change to the code, but instead make some kind of style change which affects a lot of lines, causing a lot of noise in the resulting diff. If you really wanted to change all lines of code to use 2-space indentation, you should do this in a separate pull request which does ONLY that. That way, people don't have to spend a lot of time trying to comprehend which changes you're introducing are actually meaningful, and which don't need that much attention :)